api.md

```diff
@@ -778,17 +778,37 @@ Return the same Variant object for chaining.
 ```
 
 
-#### respondWithFile(filePath)
-* ***filePath***: the path to the file to serve out (any route variables can be used in the file path as well)
+#### respondWithFile(options)
+* ***options***: options to override the default values. *(These options can also be globally provided in the smocks core options when starting the server, see ([Starting the server](#section/Concepts/Starting%2520the%2520server))*
+  * ***mockDir***: The base directory where the mocking data exists. 
+    * *default value set to* ```./mocked-data```
+  * ***fileName***: The file name smocks will search for in the ***mockDir*** directory structure and then set as the response.
+    * *default value set to* ```response.json```
+  * ***statusCode***: The status code of the response.
+    * *Default value set to* ```200```
+    
+
+This method enables you to have your json files in a route similar directory structure as follows. 
 
 Remember that using ```./``` will refer to the top level module directory (the directory where ```node_modules``` exists regardless of the location of the file that is referring to a file location with ```./```);
 
+Example 1
 ```javascript
     var smocks = require('smocks');
-    smocks.route({path: '/customer/{id}'}).respondWithFile('./customer-{id}.json')
+    smocks.route({path: '/customer/{id}'}).respondWithFile()
```

Comment:
I think this change would make things more complicated for the users when it comes to updating the data and to understand the concept, the same when it comes to involving "http method" in the path.

Using variantId.json as filename is a good idea, this will make it easier for the user to handle and updating the static data files.